apps/core/components/ItineraryCard/TripSectorWeb.js

```diff
@@ -0,0 +1,132 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { Text, StyleSheet, Icon } from '@kiwicom/universal-components';
+import { graphql, createFragmentContainer } from '@kiwicom/margarita-relay';
+import {
+  withLayoutContext,
+  LAYOUT,
+  type LayoutContextState,
+} from '@kiwicom/margarita-utils';
+
+import type { TripSectorWeb as TripSectorType } from './__generated__/TripSectorWeb.graphql';
+import Transporters from './Transporters';
+import LocalTime from './LocalTime';
+import LocationName from './LocationName';
+import { getDuration, dateFormat } from './TripSectorHelpers';
+
+type Props = {|
+  +data: ?TripSectorType,
+  +layout: number,
+|};
+
+function TripSectorWeb({ data, layout }: Props) {
+  const mobileLayout = layout < LAYOUT.largeMobile;
+  return (
+    <View style={styles.container}>
+      <Transporters data={data} />
+      <View
+        style={[
+          styles.marginEnd,
+          styles.timeDate,
+          mobileLayout && styles.timeDateMobile,
+        ]}
+      >
+        <View style={styles.row}>
+          <LocalTime data={data?.departureTime} style={styles.timeText} />
+          <Text style={styles.timeText}> - </Text>
+          <LocalTime data={data?.arrivalTime} style={styles.timeText} />
+        </View>
+        <LocalTime
+          data={data?.departureTime}
+          dateFormat={dateFormat}
+          style={styles.date}
+        />
+      </View>
+      <View style={styles.marginEnd}>
+        <Text style={styles.duration} numberOfLines={1}>
+          {getDuration(data?.duration)}
+        </Text>
+        <View style={styles.destinations}>
+          <LocationName data={data?.origin} style={styles.destination} />
+          <Icon name="route-no-stops" color="#bac7d5" size="large" />
```

Comment:
this colour could be included like `defaultTokens. colorIconTertiary`

Reply:
Thanks, updated.